app-dashboard/js/view/layers/layer-statistic/osm-polygon-statistic.js

```diff
@@ -45,23 +45,17 @@ define([
                         'geometry': that.polygon,
                         'name': polygonName
                     },
-                    null,
-                    function (data, textStatus, request) {
-                        if (data['status'] === 201) {
+                    function (data, textStatus, response) {
```

Comment:
You might want to review this part @meomancer because I changed moved the function from error callback to success callback.